frontend/server/src/main/java/org/pytorch/serve/wlm/BatchAggregator.java

```diff
@@ -77,9 +75,16 @@ public void sendResponse(ModelWorkerResponse message) {
                     throw new IllegalStateException(
                             "Unexpected job in sendResponse() with 200 status code: " + jobId);
                 }
-                if (job.getCmd() == WorkerCommands.STREAMPREDICT
-                        && prediction.getHeaders().get(TS_STREAM_NEXT).equals("true")) {
-                    jobDone = false;
+                if (jobDone) {
```

Comment:
do all the java side changes need unit tests?

Reply:
There is a tech debt in frontend which is lack of unit tests, only includes [integration test ](https://github.com/pytorch/serve/blob/master/frontend/server/src/test/java/org/pytorch/serve/ModelServerTest.java). This PR uses regression to test the e2e.